src/main/java/com/juniorstart/juniorstart/model/Technologies.java

```diff
@@ -1,20 +1,38 @@
 package com.juniorstart.juniorstart.model;
 
+import com.juniorstart.juniorstart.model.audit.UserDateAudit;
 import jakarta.validation.constraints.NotNull;
 import lombok.*;
 import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
 
 @Data
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
 @Entity
-public class Technologies {
+public class Technologies extends UserDateAudit {
```

Comment:
Czemu ta klasa jest rozszerzana

Reply:
Aby miała pola takie jak createdAt ,createdBy itp

Reply:
Nie no to wiem, tylko do jakich celów to miało by być wykorzystywane

Reply:
Nie trzeba dodawać relacji dodatkowych np user,łatwo wyjąć z bazy danych obiekty usera np findByCreatedBy